Source/UI/IdentityUI.swift

```diff
@@ -753,3 +789,45 @@ private final class DismissableNavigationController: UINavigationController {
         fatalError("init(coder:) has not been implemented")
     }
 }
+
+private final class SharedWebCredentialsController: NSObject {
+    // Stored anonymously to retain the ASAuthorizationController
+    private var authorizationController: NSObject?
```

Comment:
Let's just do `let controller = ASAuthorizationController(authorizationRequests: [ASAuthorizationPasswordProvider().createRequest()])` and don't bother clearing it out as it will be gone along with `SharedWebCredentialsController`

Reply:
We can't have a strong typed class member since it would require the entire class to be marked `@available(iOS 13.0, *)` which then cascades up. This is intentionally kept as anonymous type.